internal/serve/serve.go

```diff
@@ -419,7 +420,12 @@ func createAuthManager(dbConnectionPool db.DBConnectionPool, ec256PublicKey, ec2
 
 	passwordEncrypter := auth.NewDefaultPasswordEncrypter()
 
-	authDBConnectionPool := auth.DBConnectionPoolFromSqlDB(dbConnectionPool.SqlDB(), dbConnectionPool.DriverName())
+	ctx := context.Background()
+	dbcp, err := dbConnectionPool.SqlDB(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("getting sql db from db connection pool: %w", err)
+	}
+	authDBConnectionPool := auth.DBConnectionPoolFromSqlDB(dbcp, dbConnectionPool.DriverName())
```

Comment:
Actually, we don't need this anymore since they expect the same `DBConnectionPool`. Here we can:

1. Use the same connection pool for `auth`.
2. Remove this `DBConnectionPoolFromSqlDB` function from the `auth` package.

Reply:
Fixed in c587f7ad1fcff4d57c7e8e2bdc18f4cb18ac700b